common/persistence/nosql/nosqlplugin/cassandra/admin.go

```diff
@@ -98,34 +97,34 @@ func (db *cdb) TeardownTestDatabase() error {
 }
 
 // createCassandraKeyspace creates the keyspace using this session for given replica count
-func createCassandraKeyspace(s gocql.Session, keyspace string, replicas int, overwrite bool) (err error) {
+func (db *cdb) createCassandraKeyspace(s gocql.Session, keyspace string, replicas int, overwrite bool) (err error) {
 	// if overwrite flag is set, drop the keyspace and create a new one
 	if overwrite {
```

Comment:
no concerns here, but just FYI for all those who read this code, this is used in DB / infra provisioning and we need to be *very* careful about changes in `cadence/tools/provision` and their interaction with this.

 No concerns here though.